### DISCUSSION TITLE
Should the issue I logged in a different repo be moved to this one?

### DISCUSSION THREAD

**Discussion by StingyJack** [actions/virtual-environments#4249] [Image configuration]
https://github.com/microsoft/azure-pipelines-tasks/issues/15020  is the issue. This could be resolved by either addressing the hosted agent images or by updating the  "Use dotnet" task.  I logged the issue for the task but it should it be reported here as well?

The problem is that none of the .net core versions that are on the hosted pipeline agent are being used by the task that sets the netcore version for the tasks in the pipeline. The task looks in one place, finds nothing it needs, and the downloads the version it is told to use - every time at about 30 seconds per pipeline execution.

**Comment by miketimofeev** [comment]
@StingyJack it's a known issue and we are looking for a solution, but no ETA for now.
